stream_alert/alert_processor/output_base.py

```diff
@@ -249,17 +248,17 @@ def output_cred_name(self, descriptor):
 
         return cred_name
 
-    def format_output_config(self, config, props):
+    def format_output_config(self, service_config, values):
         """Add this descriptor to the list of descriptor this service
            If the service doesn't exist, a new entry is added to an empty list
 
         Args:
-            config [dict]: Loaded configuration as a dictionary
-            props [OrderedDict]: Contains various OutputProperty items
+            service_config [dict]: Loaded configuration as a dictionary
```

Comment:
again, let's please use [this style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/#type-annotations)